brainscore/benchmarks/lee2020.py

```diff
@@ -0,0 +1,208 @@
+            URL = {https://www.biorxiv.org/content/early/2020/07/10/2020.07.09.185116},
+            eprint = {https://www.biorxiv.org/content/early/2020/07/10/2020.07.09.185116.full.pdf},
+            journal = {bioRxiv}}'''
+
+
+class DicarloLee2020ITSpatialCorrelation(BenchmarkBase):
+
+    def __init__(self):
+        '''
+        This benchmark compares the distribution of pairwise response correlation as a function of distance between the
+            MajajHong assembly and a candidate assembly
+        Plots of this can be found in Figure 2 of the corresponding publication (see BIBTEX)
+        '''
+        super().__init__(identifier='dicarlo.Lee2020.IT-spatial_correlation',
+                         ceiling_func=None,
+                         version=0.1,
+                         parent='IT',
+                         bibtex=BIBTEX)
+
+        assembly = brainscore.get_assembly('dicarlo.MajajHong2015').sel(region='IT')
+        assembly = make_static(assembly)
+        assembly = tissue_update(assembly)
+
+        self._neuroid_reliability = InternalConsistency()(assembly.transpose('presentation', 'neuroid'))
+        self._stimulus_set = assembly.stimulus_set
+        self._target_assembly = average_repetition(assembly)
+        self._target_statistic_fn = self.create_tissue_response_correlation_target
+        self._score = SpatialCorrelationSimilarity(similarity_function=inv_ks_similarity, bin_size_mm=.1)
+
+        self.bootstrap_samples = 100_000
+        self.num_sample_arrs = 10  # number of simulated Utah arrays sampled from candidate model tissue
+        self._array_size_mm = (np.ptp(self._target_assembly.neuroid.tissue_x.data),  # physical size of Utah array in mm
+                               np.ptp(self._target_assembly.neuroid.tissue_y.data))
+
+    def __call__(self, candidate_model):
+        '''
+        This computes the statistics, i.e. the pairwise response correlation of candidate and target, respectively and
+        compute a score based on the ks similarity of the two resulting distributions
+        :param candidate_model: BrainModel
+        :return: Score, i.e. average inverted ks similarity, for the pairwise response correlation compared to the MajajHong Assembly
+        '''
+        candidate_model.start_recording(recording_target='IT', time_bins=[(70, 170)])
+        candidate_assembly = candidate_model.look_at(self._stimulus_set)
+        candidate_assembly = make_static(candidate_assembly)
+
+        candidate_statistic = []
+        for window in sample_array_locations(candidate_assembly.neuroid, self.num_sample_arrs, self._array_size_mm):
+            array_statistic = sample_response_corr_vs_dist(candidate_assembly[window],
+                                                           int(self.bootstrap_samples / self.num_sample_arrs))
+            candidate_statistic.append(array_statistic)

```

Comment:
this (together with `np.hstack(candidate_statistic)`) should go inside its own function, e.g. `sample_local_tissue_statistics` or so